pcsx2/GS/Renderers/HW/GSRendererHW.cpp

```diff
@@ -4292,7 +4294,7 @@ bool GSRendererHW::OI_GsMemClear()
 
 				for (int x = r.left; x < r.right; x++)
 				{
-					*pa.value(x) = 0; // Here the constant color
+					*pa.value(x) = color; // Here the constant color
```

Comment:
Comment refers to the colour as a constant, which is no longer the case relative to the guest, as it's taken from RGBAQ.

Reply:
it is constant, I set it constant above, it doesn't change through the loop :)